arangodb3/README.md

```diff
@@ -0,0 +1,93 @@
+<!--
+Copyright (c) 2012 - 2015 YCSB contributors. All rights reserved.
```

Comment:
Since these are new files can they just be 2017? Should be the same for the other files too.